### PR TITLE
Automation - Hybrid Runbook Worker Group operations related cmdlets added

### PR DESCRIPTION
**Pull request by krmanupa**
## Description

Created new cmdlets for the Hybrid Worker Group related operations on Automation Resource. 
All the automation cmdlets have now been updated with the private autorest sdk, 

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by jagpreetstamber — APPROVED** [review]

File: src/Automation/Automation/help/Az.Automation.md [hunk 1]
```diff
@@ -185,8 +200,11 @@ Removes DSC configurations from Automation.
 ### [Remove-AzAutomationDscNodeConfiguration](Remove-AzAutomationDscNodeConfiguration.md)
 Removes metadata from DSC node configurations in Automation.
 
-### [Remove-AzAutomationHybridWorkerGroup](Remove-AzAutomationHybridWorkerGroup.md)
-Removes hybrid worker group from Automation.
+### [Remove-AzAutomationHybridRunbookWorker](Remove-AzAutomationHybridRunbookWorker.md)
+{{ Fill in the Synopsis }}
```
Comment (isra-fel) [inline src/Automation/Automation/help/Az.Automation.md:204]: Please fill in the placeholders in this file

**Review by isra-fel — CHANGES_REQUESTED** [review]
1. `Get-AzAutomationHybridWorkerGroup` and `Remove-AzAutomationHybridWorkerGroup` were removed. This is a breaking change and cannot be merged until the next release.
2. Please fix failed test cases - they likely need to be recorded again
3. Please fix the issues with examples. See the log of CI for more details.

**Comment by isra-fel** [comment]
Like I said, removing cmdlets is a breaking change. Please submit a new PR to do it so that it won't block this one.

**Comment by wyunchi-ms** [comment]
The release branch is already released. Please retarget your PR to `main` branch.

**Review by wyunchi-ms — APPROVED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**